CONTRIBUTING.md

```diff
@@ -63,55 +63,35 @@ Then maybe someone will take over and implement your use case… or maybe you wi
 Even better, complete the migration tool implementation to support the missing case!
 
 After you have added the missing case (see previous section), run the following
-sbt task (with sbt started from the `scalafix/` directory) to run the
-migration tool on the input files and check whether the result matches the
+sbt task to run the migration tool on the input files and check whether the result matches the
 expected output files:
 
 ~~~
 > scalafix-tests/test
 ~~~
 
-Fix the implementation of the rule (in the
-`rules/src/main/scala/fix/NewCollections.scala` file) until the
+Fix the implementation of the rule (in the `rules/src/main/scala/fix/NewCollections.scala` file) until the
 tests are green. You can find more help about the scalafix API in its
 [documentation](https://scalacenter.github.io/scalafix/docs/rule-authors/setup).
 
 
 ### Scalafix Teskit Directory Layout
 
+
 ```
-.
-├── data                    |
-│   └── src                 |
-│       └── main            |
-│           └── scala       | Project to avoid duplicating code between input and output
-├── input                   |
-│   └── src                 |
-│       └── main            |
-│           ├── scala       | Input that cross-compile
-│           └── scala-2.12  | 2.12 specific input
-├── output                  |
-│   └── src                 |
-│       └── main            |
-│           └── scala       | Output that cross-compile
-├── output212               |
-│   └── src                 |
-│       └── main            |
-│           └── scala-2.12  | 2.12 specific output
-├── output213               |
-│   └── src                 |
-│       └── main            |
-│           └── scala       | 2.13 specific output (from a cross-compiled input)
-├── output213-failure       |
-│   └── src                 |
-│       └── main            |
-│           └── scala       | 2.13 specific output that cannot be migrated due to technical limitations
-├── rules                   |
-│   └── src                 |
-│       └── main            |
-│           └── scala       | Rule implementations
-└── tests                   |
-    └── src                 |
-        └── test            |
-            └── scala       | Scalafix testkit launcher (useful to run a single input file)
++------------------+-----+-----------------+
+|                  |Input|     Output      |
+|                  | 2.12| 2.11| 2.12| 2.13|
++------------------+-----+-----+-----+-----+
+|data              |  X  |  X* |  X  |  X  |
```

Comment:
What does the star mean?

Reply:
The only exception is in this next PR: https://github.com/scala/scala-collection-compat/pull/122/files#diff-305e21c374e13a65d851eaf004df6a39